paymentapi/src/main/java/fi/hel/verkkokauppa/payment/logic/context/PaytrailPaymentContext.java

```diff
@@ -0,0 +1,67 @@
+package fi.hel.verkkokauppa.payment.logic.context;
+
+public class PaytrailPaymentContext {
```

Comment:
Could you use `@Data` annotation to get rid of getters/setters?

Reply:
Good point, I'll use that instead.